sklearn/multioutput.py

```diff
@@ -0,0 +1,151 @@
+"""
+This module implements multioutput regression.
```

Comment:
Hi, sorry for commenting here but just wanted to inform that #6127 also adds the MultiOutputClassifier to a separate module 'multioutput.py'